README.md

```diff
@@ -103,6 +103,7 @@ See [BUILDING.md](BUILDING.md)
 * Cmd/Ctrl-M: change *m*onitor
 * Cmd/Ctrl-F: toggle *f*ull screen
 * Mouse Scroll Up / Down: next or previous preset (hard transition)
+* Return: search for preset (RETURN or ESCAPE to exit search)
```

Comment:
maybe `/` too, vim-style?

Reply:
I started with this but gave up because of how SDL_StartTextInput will still grab the / key. I am likely missing something, but this could be a wish list.